examples/todoapp/src/App.tsx

```diff
@@ -32,10 +33,28 @@ function App() {
             allTodos
               @fabrixView(
                 input: [
-                  { field: "collection", config: { label: "Your tasks" } }
+                  { field: "collection", config: { label: "タスク一覧" } }
+                  { field: "collection.id", config: { hidden: true } }
+                  { field: "collection.hasDone", config: { label: "Status" } }
+                  {
+                    field: "collection.actions"
```

Comment:
What is the `collection.actions`? Is this a reserved word?

Reply:
No, any word can be usable for virtual fields. Does it make sense using underscore, then?

```
  {
    field: "collection.__actions"
    config: {
      # ...
    }
  }
```

Reply:
Why not simply use “actions”?
`collection.` makes it look like a PATH.

Reply:
Starting with `collection.*` tells fabrix to add a new virtual column, so if specifying `actions` instead of `collection.actions`, fabrix does not read it as an virtual column. I think this behaviour should be documented.

Reply:
Is it possible for something that doesn't match `collection.*` to be assigned to a `field`?

Reply:
Yes, that's the way how virtual fields work.